src/lib/snack-bar/snack-bar-ref.ts

```diff
@@ -11,6 +11,12 @@ import {Observable} from 'rxjs/Observable';
 import {Subject} from 'rxjs/Subject';
 import {MatSnackBarContainer} from './snack-bar-container';
 
+/** Event that is emitted when a snack bar is dismissed. */
+export interface MatSnackBarDismiss {
+  /** Whether the snack bar was dismissied using the action button. */
+  dismissedByAction: boolean;
```

Comment:
Should this be `closedByAction` since the method that would cause this to be true is `closeWithAction()`?

Reply:
That's true, but the method that returns is called `afterDismissed`. Either way it would be slightly confusing.

Reply:
Maybe we should mark one of the terms as deprecated to fix it long term?  I am good with either wording being removed, just as long as we create consistency.

Reply:
My vote is for removing the "dismissed" wording and calling it "close" (also renaming the `dismiss` method). That way it's consistent with the dialog as well.

Reply:
SGTM